client/src/pages/Calendar.js

```diff
@@ -22,7 +23,18 @@ class Calendar extends Component {
 
   render() { 
     return ( <div>
+      <Hero/>
```

Comment:
super nitpicky but could you fix the indent and spacing issues here? the <div> next to return should be on a new line

Reply:
No problem, I appreciate your thoroughness.  Done